src/content-layout/__integ__/content-layout.test.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+import styles from '../../../lib/components/content-layout/styles.selectors.js';
+
+class ContentLayoutPageObject extends BasePageObject {
+  async isOverlapEnabled() {
+    const backgroundElement = await this.browser.$(`.${styles.background}`);
+    const classes = await backgroundElement.getAttribute('class');
+    return !classes.includes(styles['is-overlap-disabled']);
+  }
+
+  async toggleHeaderPresent() {
+    const headerPresentCheckbox = await this.browser.$('#header-present-checkbox');
+    await headerPresentCheckbox.click();
+  }
+}
+
+describe('ContentLayout component', () => {
+  test(
+    'It renders the overlap by default',
```

Comment:
very minor: I believe test names should start lower-case. Actually, you probably want to remove the "It" entirely, so it reads like a sentence. That's why the `it` synonym for `test` exists. For example, `it("renders the overlap by default")`